app/models/katello/content_view_version.rb

```diff
@@ -41,7 +41,26 @@ class ContentViewVersion < Katello::Model
     scope :default_view, joins(:content_view).where("#{Katello::ContentView.table_name}.default" => true)
     scope :non_default_view, joins(:content_view).where("#{Katello::ContentView.table_name}.default" => false)
 
-    scoped_search :on => [:content_view_id, :major, :minor]
+    scoped_search :on => :content_view_id
+    scoped_search :on => :major, :rename => :version, :complete_value => true, :ext_method => :find_by_version
+    scoped_search :in => :repositories, :on => :name, :rename => :repository, :complete_value => true
+
+    def self.find_by_version(_key, operator, value)
+      if ['>', '<', '=', '<=', '>=', "<>", "!=", 'IN', 'NOT IN'].include?(operator) && value.to_f >= 1
+        major, minor = value.split(".")
+        case
+        when /[<>]/ =~ operator
+          minor ||= 0
+          query = where("major #{operator} :major OR (major = :major AND minor #{operator} :minor)", :major => major, :minor => minor)
+        when minor.nil?
+          query = where("major #{operator} (:major)", :major => major)
+        else
+          query = where("major #{operator} (:major) and minor #{operator} (:minor)", :major => major, :minor => minor)
+        end
+        _, conditions = query.to_sql.split("WHERE")
+      end
+      { :conditions => conditions || ""}
```

Comment:
You could initialize conditions to empty string at the top of the function and simplify this declaration.